lib/radiator/directory/editor/editor.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Radiator.Directory.Editor.Editor do
```

Comment:
`Editor.Editor` is very confusing. I understand the naming -- both editors are actually different -- but it feels time to come up with a  better name for `Radiator.Directory.Editor` as _this_ module is appropriately named `Editor`.

I've seen `CMS` used for something like this somewhere but it opens up another confustion for us because of the already existing `radiator-cms`. `Radiator.Directory.EditorAsInTheSoftware.EditorAsInThePerson` ¯\\_(ツ)_/¯